src/mmw/js/src/modeling/models.js

```diff
@@ -751,7 +751,7 @@ var ScenarioModel = Backbone.Model.extend({
                     var unmodifiedFetchPromises = fetchResults(true);
                     return $.when(unmodifiedFetchPromises.startPromise,
                                   unmodifiedFetchPromises.pollingPromise)
-                            .then(function() {
+                            .done(function() {
```

Comment:
Uhh sorry I think we should keep the `.then`. `.done` would return the parent promise, whereas `.then` returns a new promise. See https://stackoverflow.com/questions/5436327/jquery-deferreds-and-promises-then-vs-done